src/common/utils/registry.h

```diff
@@ -378,6 +378,12 @@ namespace registry
                     std::wstring kindMapPath = L"Software\\Microsoft\\Windows\\CurrentVersion\\Explorer\\KindMap";
                     changes.push_back({ HKEY_LOCAL_MACHINE, kindMapPath, fileType, fileKindType, false});
                 }
+                if (fileType == L".reg")
```

Comment:
maybe also check/assert that `handlerType == PreviewHandlerType::preview`

Reply:
good point. done